src/pages/HomePage.js

```diff
@@ -12,7 +12,7 @@ export default function HomePage() {
         </LinkStyled>
       </header>
       <main>
-        <h1>Your entries:</h1>
+        <h1>Tight lines, Caspar</h1>
```

Comment:
Good title for a h1. I would recommend to keep that.

Reply:
Thanks a lot